.github/workflows/build-and-release.yml

```diff
@@ -110,6 +110,10 @@ jobs:
           mv e2e.test.exe e2e.test-windows-amd64.exe
       - name: Upload dist artifact
         uses: actions/upload-artifact@v2
+        run: |
+          sha256sum kluctl-linux-amd64 > checksums.txt
+          sha256sum kluctl-darwin-amd64 >> checksums.txt
+          sha256sum kluctl-windows-amd64.exe >> checksums.txt
```

Comment:
Unfortunately, this will not work as this job is only executed for a specific os/arch and thus the other ones are missing. The sum calculation must be done in the release job right after downloading the artifacts (which then have all os/arch versions)